src/core/reducers/addons.js

```diff
@@ -1,15 +1,36 @@
 /* @flow */
 import { ADDON_TYPE_THEME } from 'core/constants';
 import type { ErrorHandlerType } from 'core/errorHandler';
-import type { AddonType } from 'core/types/addons';
+import type { AddonType, ExternalAddonType } from 'core/types/addons';
 
 
+export const ADDONS_LOADED = 'ADDONS_LOADED';
```

Comment:
Can we have `LOAD_ADDON` instead? See: https://github.com/mozilla/addons-frontend/pull/3025#discussion_r135909932

Reply:
ah, right, thanks